includes/admin/payments/orders.php

```diff
@@ -749,10 +841,14 @@ function edd_order_details_extras( $order = false ) {
 
 				<?php if ( ! edd_is_add_order_page() && ! empty( $order->id ) ) : ?>
 					<?php if ( ! empty( $order->gateway ) ) : ?>
-					<div class="edd-order-gateway edd-admin-box-inside edd-admin-box-inside--row">
-						<span class="label"><?php esc_html_e( 'Gateway', 'easy-digital-downloads' ); ?></span>
-						<span class="value"><?php echo edd_get_gateway_admin_label( $order->gateway ); ?></span>
-					</div>
+						<div class="edd-order-gateway edd-admin-box-inside edd-admin-box-inside--row">
+							<div class="edd-form-group">
+								<label for="edd_gateway" class="edd-form-group__label"><?php esc_html_e( 'Gateway', 'easy-digital-downloads' ); ?></label>
+								<div class="edd-form-group__control">
+									<input class="edd-form-group__input" id="edd_gateway"><?php echo edd_get_gateway_admin_label( $order->gateway ); ?>/>
+								</div>
+							</div>
```

Comment:
The `edd-form-group` needs to be around the `<?php echo $gateway_select; ?>` and `label` below. The markup here is for viewing an existing Order, so the Gateway shouldn't be editable.

When viewing an Order let's make "Gateway" and "IP" appear the same as the rows in the "Order Amounts" box.